packages/core/core/src/InternalAsset.js

```diff
@@ -241,6 +354,33 @@ export default class InternalAsset {
     }
   }
 
+  async getAST(): Promise<?AST> {
+    if (this.value.astKey != null) {
```

Comment:
So if you call getAST multiple times, this would repeatedly deserialize it from cache?

Reply:
Yeah, this is true of `getMap` as well. We should address this.